dali/kernels/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@
 
 project(dali_kernel)
 
+add_subdirectory(audio)
```

Comment:
I'd suggest renaming to "signal", since FFT can be used other kinds of data, too.

Reply:
done